app/server/src/main/scala/org/bitcoins/server/WalletRoutes.scala

```diff
@@ -38,6 +39,17 @@ case class WalletRoutes(wallet: UnlockedWalletApi, node: Node)(
     Server.httpSuccess(sendString)
   }
 
+  private def handleBroadcastable(
+      tx: Transaction,
+      noBroadcast: Boolean): String = {
+    if (noBroadcast) {
+      tx.hex
+    } else {
+      node.broadcastTransaction(tx)
+      tx.txIdBE.hex
+    }
+  }
```

Comment:
have we decided to get rid of those sweep txs then? I think that's a good idea but we should make sure to do that in the `noBroadcast` case as well

Reply:
Those are still handled, `handleBroadcastable` is just called for both transactions

Reply:
I see, well in this case I actually think we should take this opportunity (or the next PR if you prefer) to get rid of the unnecessary sweep txs. We shouldn't be broadcasting frivolous txs by default :)

Reply:
I think that is more fitting for a separate PR

Reply:
could you open an issue?